docs/source/en/generation_strategies.md

```diff
@@ -77,7 +77,7 @@ producing highly repetitive results.
 You can override any `generation_config` by passing the parameters and their values directly to the [`generate`] method:
 
 ```python
->>> my_model.generate(**inputs, num_beams=4, do_sample=True)
+>>> my_model.generate(**inputs, num_beams=4, do_sample=True)  # doctest: +SKIP
```

Comment:
skip: this is a dummy, representative line

Reply:
What's the error if we don't skip?

Reply:
IIRC the purpose, we can use `# doctest: +IGNORE_RESULT` instead of `# doctest: +SKIP`

Reply:
In this case it won't run at all, since `my_model` isn't defined :D

Reply:
Got it, thanks